src/legacy/core_plugins/embeddable_api/public/panel/_embeddable_panel.scss

```diff
@@ -0,0 +1,125 @@
+
+.embPanel {
+  z-index: auto;
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  position: relative;
+
+  // SASSTODO: The inheritence factor stemming from embeddables makes this class hard to change
+  .panel-content {
+    display: flex;
+    flex: 1 1 100%;
+    z-index: 1;
+    min-height: 0; // Absolute must for Firefox to scroll contents 
+    overflow: auto;
```

Comment:
This line seems to be causing issues with scrollbars appearing and disappearing. It wasn't in the original dashboard styles. Any reason why it's necessary here?

<img src="https://d.pr/free/i/0yuh04+" width="50%" />

Reply:
It prevented this behavior, though you can't tell on this PR because there are none of the V2 Embeddables on a dashboard with resizeable panels.

<img width="599" alt="Screen Shot 2019-06-04 at 6 02 30 PM" src="https://user-images.githubusercontent.com/16563603/58916899-fc067500-86f2-11e9-84c3-aae28a40f4c1.png">

I suppose I can leave out for now and put back in when I integrate with the rest of dashboard.  It wasn't needed in original dashboard code because this error message looked totally different.  I switched it the EUICallout.  Maybe when I tackle the other version I can go back to the original error and then not need this extra line.